snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -579,65 +576,38 @@ func (b *Bootstrapper) markUnavailable(nodeID ids.NodeID) {
 // If [blk]'s height is <= the last accepted height, then it will be removed
 // from the missingIDs set.
 func (b *Bootstrapper) process(ctx context.Context, blk snowman.Block, processingBlocks map[ids.ID]snowman.Block) error {
+	lastAcceptedID, err := b.VM.LastAccepted(ctx)
+	if err != nil {
+		return fmt.Errorf("couldn't get last accepted ID: %w", err)
+	}
+	lastAccepted, err := b.VM.GetBlock(ctx, lastAcceptedID)
+	if err != nil {
+		return fmt.Errorf("couldn't get last accepted block: %w", err)
+	}
+	lastAcceptedHeight := lastAccepted.Height()
```

Comment:
We could optimize this a bit by tracking the last accepted height ourselves... But I found this more obviously correct.